db/migrate/20220518005355_add_user_ref_to_inventory.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToInventory < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :inventories, :user, null: false, foreign_key: true
+  end
+end
```

Comment:
- [ ] Since, we already have a reference for the user in the inventory table create migration, I don't think this is necessary. Kindly rectify.